azure-pipelines-with-e2etests.yml

```diff
@@ -3,8 +3,12 @@
 # And then running E2E tests and publishing them.
 
 trigger:
-  - '2.0-preview'
-  - 'release*'
+  branches:
+    include:
+      - '2.0-preview'
+  tags:
+    include:
+      - v2.0.0-beta.*
```

Comment:
So the steps are: 

1. prerelease (creates release branch, makes changes to integrity hash and version constants, creates PR with changelog contents against release branch), 
2. postrelease (creates GitHub (pre)Release, creates tag, creates PR against 2.0-preview), 
3. e2e & script tag tests trigger on tag creation from postrelease step, 
4. CI build triggers on tag creation from postrelease step, which triggers publishing internally. 

Did I summarize that correctly?

Reply:
Yes!